File: WebApp/Scripts/app/model/PhSmallMail.js
Ext.define('WEBAPP.model.PhSmallMail', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'SEND_TO', type: 'string' },
        { name: 'MAIL_ADD', type: 'string' },
        { name: 'MEMO', type: 'string' }
    ]
});

File: WebApp/Models/PH/PH_SMALL_MAIL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.Models
{
    public class PH_SMALL_MAIL : JCLib.Mvc.BaseModel
    {
        public string SEND_TO { get; set; }
        public string MAIL_ADD { get; set; }
        public string MEMO { get; set; }
    }
}